django-server/server/admin/static/admin/stats.js

```diff
@@ -0,0 +1,873 @@
+        guideline.value(1);
+        plots.push(guideline);
+      }
+
+      const group = new Plottable.Components.Group(plots);
+      chart = new Plottable.Components.Table([
+        [yLeftAxis, group, yRightAxis],
+        [null, xAxis, null]
+      ]);
+      chart.renderTo('#graph-contents');
+
+      this.update_header();
+    }
+  },
+  mounted: function() {
+    this.change_grouping();
+
+    let app = this;
+    loadCafeterias()
+      .then(() => app.showChart())
+      .catch(error => app.showError(error.message));
+  }
+});
+
+function loadCafeterias() {
+  return fetch(API_URL_BASE + 'cafeterias/?fields=id,name&owner_id=' + USER_ID)
+    .then(response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw new Error('HTTP status ' + response.status);
+      }
+    })
+    .then(data => {
+      let result = {};
+      data.forEach(cafeteria => {
+        cafeteria.color = colorMaker.nextColor();
+        result[cafeteria.id] = cafeteria;
+      });
+      return result;
+    })
+    .then(data => {
+      Vue.set(App, 'cafeterias', data);
+    });
+}
+
+function makeOccupancyData(startDate, endDate) {
+  const currentDate = new Date();
+  startDate = startDate || new Date();
+  var startYear = startDate.getFullYear();
+  var startMonth = startDate.getMonth();
+  var startDay = startDate.getDate();
+  var toReturn = [];
+  let i = 0;
+  while (true) {
+    const timestamp = new Date(startDate);
+    timestamp.setHours(startDate.getHours() + i);
+    if (timestamp >= endDate || timestamp >= currentDate) {
+      break;
+    }
+
+    let people_in = Math.round(Math.max(i > 0 ? toReturn[i - 1].value.total + Math.random() * 10 - 5 : Math.random() * 5, 0));
+    toReturn[i] = {
+      timestamp: timestamp,
+      value: {
+        relative: people_in / 50,
+        total: people_in
+      }
+    };
+
+    i++;
+  };
+  return toReturn;
+}
+
+function loadOccupancy(cafeteria_id, from, to) {
+  let sample_data = [
+    {
+      timestamp: '2020-05-20T09:37:38.593Z',
+      value: {
+        relative: 0.86, // 86% occupied
+        total: 43       // 43 people inside
+      }
+    },
+    {
+      timestamp: '2020-05-20T09:38:38.593Z',
+      value: {
+        relative: 0.98,
+        total: 49
+      }
+    },
+    {
+      timestamp: '2020-05-20T09:39:38.593Z',
+      value: {
+        relative: 0.90,
+        total: 45
+      }
+    },
+  ];
+
+  let sample_promise = new Promise(function(resolve, reject) {
+    setTimeout(() => {
+      resolve(makeOccupancyData(from, to));
+    }, DEBUGGING_LAG);
+  });
+
+  // TODO: Replace sample_promise with actual API call when implemented
+  /*
+  fetch(API_URL_BASE + 'cafeterias/' + d + '/occupancy/?from=' + from.toISOString() + '&to=' + to.toISOString())
+    .then(response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw new Error('HTTP status ' + response.status);
+      }
+    })
+  */
```

Comment:
make an issue for this maybe not to forget

Reply:
Referencing #141 here so it shows up in there.